packages/office-ui-fabric-react/src/components/TeachingBubble/TeachingBubbleContent.base.tsx

```diff
@@ -89,19 +89,31 @@ export class TeachingBubbleContentBase extends BaseComponent<ITeachingBubbleProp
     if (headline) {
       headerContent = (
         <div className={classNames.header}>
-          <p className={classNames.headline} id={ariaLabelledBy}>
-            {headline}
-          </p>
+          {typeof children === 'string' ? (
```

Comment:
how about const wrapperAs = typeof children === 'string' ? 'p' : 'div'; to avoid duplication below.

Reply:
i suppose yes :)